src/platform/i386/Makefile

```diff
@@ -5,7 +5,7 @@ LD := ld -m elf_i386
 AS := as --32 -g
 
 INCPATH := ../../kernel/include
-CFLAGS := -g3 -ffreestanding -Wno-unused-function -Wno-unused-variable -Wno-unused-but-set-variable -nostdlib -mno-red-zone -I$(INCPATH)
+CFLAGS := -g3 -O3 -ffreestanding -Wno-unused-function -Wno-unused-variable -Wno-unused-but-set-variable -nostdlib -mno-red-zone -I$(INCPATH)
```

Comment:
We need -g for symbols. If you remember, we were not able to use objdump to get C code inlined with the assembly dump. So I kept both. I had initially set to only -O3, thinking that it would slow down if there are symbols but later after discussing with you, I kept both, because symbols are useful and -g won't affect the performance.